lua/telescope/make_entry.lua

```diff
@@ -612,31 +612,41 @@ function make_entry.gen_from_packages(opts)
   end
 end
 
-function make_entry.gen_from_apropos()
+function make_entry.gen_from_apropos(opts)
+  local sections = {}
+  for _, section in ipairs(vim.split(opts.sections, ',', true)) do
+    sections[section] = true
+  end
```

Comment:
Hmmm. I thought we can just pass in `sections = { 1, 5 }` but i only thought about the lua side. I am not sure if i like a comma separated string

Reply:
I wrote this for launching from commands such as `:Telescope man_pages sections=1,5`. Ideally, `convert_user_opts()` should parse these comma-separated values into Lua tables.

I will file an issue for this.

Reply:
Issued in #447 and will be fixed in #450.

54f68cf I've removed this logic to expect #450 to be merged.